website/content/partials/faq/client-count/upgrading.mdx

```diff
@@ -16,6 +16,8 @@ provided in the Vault UI and API.
 | 1.13    | UI        | Combined current month and previous history into one dashboard
 | 1.16    | All       | Added synced secrets as a new category in client counting
 | 1.17    | All       | Separated PKI ACME certificates as their own category in client counting
+| 1.20    | All       | Get client count endpoint response excludes the current month and any future date
+| 1.20+ent    | All       | Actual values are returned for `new_clients` counts for the current month and they are no longer an estimate
```

Comment:
@schavis is this formatting ok? And is it alright that we call it 1.20 and 1.20+ent?

Reply:
Added edition column